$OpenBSD: patch-Source_Files_Sound_SoundManager_cpp,v 1.2 2017/05/24 01:01:29 espie Exp $

Directly include iostream for cerr (newer Boost versions no longer pull it in for us)
upstream rev 5012

Index: Source_Files/Sound/SoundManager.cpp
--- Source_Files/Sound/SoundManager.cpp.orig
+++ Source_Files/Sound/SoundManager.cpp
@@ -22,6 +22,7 @@ SOUND.C
 
 #include <iostream>
 
+#include <iostream>
 #include "SoundManager.h"
 #include "ReplacementSounds.h"
 #include "sound_definitions.h"
